projects/plugins/wpcomsh/wpcomsh.php

```diff
@@ -0,0 +1,650 @@
+ * p9o2xV-2zY-p2
+ */
+add_filter( 'default_option_jetpack_sso_match_by_email', '__return_true' );
+
+/**
+ * Admin enqueue style
+ */
+function wpcomsh_admin_enqueue_style() {
+	wp_enqueue_style(
+		'wpcomsh-admin-style',
+		plugins_url( 'assets/admin-style.css', __FILE__ ),
+		array(),
+		WPCOMSH_VERSION
+	);
+}
+add_action( 'admin_enqueue_scripts', 'wpcomsh_admin_enqueue_style', 999 );
+
+/**
+ * Allow custom wp options
+ *
+ * @param array $options The options.
+ *
+ * @return array
+ */
+function wpcomsh_allow_custom_wp_options( $options ) {
+	// For storing AT options.
+	$options[] = 'at_options';
+	$options[] = 'at_options_logging_on';
+	$options[] = 'at_wpcom_premium_theme';
+	$options[] = 'jetpack_fonts';
+	$options[] = 'site_logo';
+	$options[] = 'footercredit';
+
+	return $options;
+}
+add_filter( 'jetpack_options_whitelist', 'wpcomsh_allow_custom_wp_options' );
+
+add_filter( 'jetpack_site_automated_transfer', '__return_true' );
+
+/**
+ * Check site has pending automated transfer
+ *
+ * @return bool
+ */
+function check_site_has_pending_automated_transfer() {
+	return get_option( 'has_pending_automated_transfer' );
+}
+
+add_filter( 'jetpack_site_pending_automated_transfer', 'check_site_has_pending_automated_transfer' );
+
+/**
+ * Require library helper function
+ *
+ * @param mixed $slug Slug of library.
+ * @return void
+ */
+function require_lib( $slug ) {
+	if ( ! preg_match( '|^[a-z0-9/_.-]+$|i', $slug ) ) {
+		return;
+	}
+
+	$basename = basename( $slug );
+
+	$lib_dir = __DIR__ . '/lib';
+
+	/**
+	 * Filter the location of the library directory.
+	 *
+	 * @since 2.5.0
+	 *
+	 * @param string $lib_dir Path to the library directory.
+	 */
+	$lib_dir = apply_filters( 'require_lib_dir', $lib_dir );
+
+	$choices = array(
+		"$lib_dir/$slug.php",
+		"$lib_dir/$slug/0-load.php",
+		"$lib_dir/$slug/$basename.php",
+	);
+	foreach ( $choices as $file_name ) {
+		if ( is_readable( $file_name ) ) {
+			require_once $file_name;
+			return;
+		}
+	}
+}
+
+/**
+ * Provides a fallback Google Maps API key when otherwise not configured by the
+ * user. This is subject to a usage quota.
+ *
+ * @see p5j4vm-1gT-p2
+ *
+ * @param string $api_key Google Maps API key.
+ * @return string Google Maps API key
+ */
+function wpcomsh_google_maps_api_key( $api_key ) {
+	// Fall back to the dotcom API key if the user has not set their own.
+	return ( empty( $api_key ) ) ? 'AIzaSyCq4vWNv6eCGe2uvhPRGWQlv80IQp8dwTE' : $api_key;
+}
+add_filter( 'jetpack_google_maps_api_key', 'wpcomsh_google_maps_api_key' );
+
+/**
+ * We have some instances where `track_number` of an audio attachment is `??0` and shows up as type string.
+ * However the problem is, that if post has nested property attachments with this track_number, `json_serialize` fails silently.
+ * Of course, this should be fixed during audio upload, but we need this fix until we can clean this up properly.
+ * More detail here: 235-gh-Automattic/automated-transfer
```

Comment:
No need to shorthand this
```suggestion
 * More detail here: https://github.com/Automattic/automated-transfer/issues/235
```